prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,11 @@
+var languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log('languages array', languages);
+
+var firstElement = 'html';
+console.log('the first element of the array is', firstElement);
+var thirdElement = 'javascript';
+console.log('the 3rd element of the array is', thirdElement);
```

Comment:
You can get an element from an array by using its index.

```js
var firstElement = languages[0];
```

Reply:
Great thank you I fixed it. Im not sure how my objects assignment ended up in this folder, what is the best way to remove it?

Reply:
That's ok, it usually happens when you `git add` more files than you intend to when before committing.